software/openvisualizer/bin/openVisualizerApp/openVisualizerApp.py

```diff
@@ -58,6 +61,16 @@ def __init__(self,confdir,datadir,logdir,simulatorMode,numMotes,trace,debug,simT
             self.simengine        = SimEngine.SimEngine(simTopology)
             self.simengine.start()
         
+        # import the number of motes from json file given by user (if the importTopo option is enabled)
+        if self.pathTopo and self.simulatorMode and os.path.exists(pathTopo):
```

Comment:
with the try/except structure below, `and os.path.exists(pathTopo)` is not needed